src/service/getVideos.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+async function getVideos() {
+  const response = await axios({
+    method: 'GET',
+    url: '/api/videos',
+    headers: {
+      Accept: 'application/json',
+    },
+  });
+  return response.data.videos;
+}
+
+export { getVideos };
```

Comment:
All the services files could have the extension of `name.service.js`. Why so? It looks cool in VS-Code and on github😂. Although I also don't know the advantage of using the `.service.js` extension but it is worth using.

Reply:
Ok, I will change the naming convention.